src/main/java/uk/gov/hmcts/reform/judicialbooking/domain/model/Booking.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.reform.judicialbooking.domain.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.ZonedDateTime;
+import java.util.UUID;
+
+@Builder
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class Booking {
+
+    private UUID id;
+    private String userId;
+    private String appointmentId;
+    private String roleId;
+    private String contractTypeId;
+    private String baseLocationId;
```

Comment:
we don't need to maintain separate model class for booking. try reusing BookingEntity instead.